lib/PuppeteerSharp/Frame.cs

```diff
@@ -12,11 +12,12 @@ public class Frame
         private Frame _parentFrame;
         private string _defaultContextId = "<not-initialized>";
         private object _context = null;
-        private string _url = string.Empty;
-        private List<WaitTask> _waitTasks;
+        private string _url = string.Empty;        
         private bool _detached;
         private TaskCompletionSource<ElementHandle> _documentCompletionSource;
 
+        internal List<WaitTask> WaitTasks;
```

Comment:
This should be expressed as a getter property right?
```
internal List<WaitTask> WaitTask { get; }
```

Reply:
yup